README.md

```diff
@@ -6,15 +6,25 @@ You can find us on Twitter [@MozillaReality](https://twitter.com/mozillareality)
 
 [![Task Status](https://github.taskcluster.net/v1/repository/MozillaReality/FirefoxReality/master/badge.svg)](https://github.taskcluster.net/v1/repository/MozillaReality/FirefoxReality/master/latest) [Build results](https://github.taskcluster.net/v1/repository/MozillaReality/FirefoxReality/master/latest)
 
-## Bleeding-edge developer APKs
+## Download developer APKs (debug builds)
```

Comment:
this heading reads awkwardly IMO. any suggestions?

Reply:
It's OK. The reason I think it's worth mentioning it explicitly is performance may be different for debug binaries, based on profiling and log code, optimization disabled etc.

Reply:
From what I can tell these links are pointing to staging builds which are release builds, the only difference is they are signed with the staging key instead of the the release key and thus can not be used in App Stores. But they are fully optimized builds. We do also build debug version of each APK but these link do not seem to be pointing to them.
For example:
```
public/FirefoxReality-1.1.4-31281846-googlevr-arm-debug.apk
public/FirefoxReality-1.1.4-31281846-googlevr-arm-staging-signed-aligned.apk
```
The first apk is the debug build for GoogleVR and the second APK is the full release build signed with the staging key.